msrcrypto.js

```diff
@@ -9512,8 +9511,15 @@ var publicMethods = {
         ///     <returns type="ArrayBufferView">Returns ArrayBufferView if supported.</returns>
         /// </signature>
 
-        var i;
-        var randomValues = msrcryptoPseudoRandom.getBytes(array.length);
+        var i, prngCrypto, randomValues;
+
+        prngCrypto = window.crypto || window.msCrypto;
+        if (typeof module !== 'undefined') {

```

Comment:
this node check might fail if someone has defined module in an upper scope. SO suggests this as a slight improvement: http://stackoverflow.com/a/11918368